core/src/main/java/io/grpc/MethodDescriptor.java

```diff
@@ -54,6 +54,7 @@
   private final Marshaller<ReqT> requestMarshaller;
   private final Marshaller<RespT> responseMarshaller;
   private final boolean idempotent;
+  private final boolean cacheable;
```

Comment:
I think the name was supposed to be "safe" (following HTTP's method semantics). We should coordinate with @makdharma.